### PR TITLE
Fix Udacity rule.

### PR DESCRIPTION
**Pull request by bgkonzman**
See issue #4187

The securecookie rule was preventing sso login. 

Driveby update: fix the "not secured by server" section to reflect
that the DgU00 cookie is now secure.

### CODE REVIEW

File: src/chrome/content/rules/Udacity.com.xml [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
 	<!--	Not secured by server:
 					-->
-	<!--securecookie host="^www\.udacity\.com$" name="^(DgU00|UY-VISITOR-ID)$" /-->
+	<!--securecookie host="^www\.udacity\.com$" name="^(UY-VISITOR-ID)$" /-->
 
-	<securecookie host="^www\.udacity\.com$" name=".+" />
+	<securecookie host="^\.udacity\.com$" name=".+" />
```
Comment (J0WI) [inline src/chrome/content/rules/Udacity.com.xml]: Could you also simplify this redirect?

**Comment by bgkonzman** [comment]
That should do it. Of course, let me know if I'm wrong. Thanks, J0WI.

**Comment by J0WI** [comment]
Works for me:
discussions.udacity.com
forums.udacity.com
profiles.udacity.com
review.udacity.com
ar.udacity.com
email.udacity.com (only some deeplinks)

Mismatch:
blog.udacity.com
status.udacity.com
career.udacity.com

Refused:
labs.udacity.com

**Comment by bgkonzman** [comment]
OK, will work on it later today.

I'm new to the project. Is there something I should read other than what's on the EFF website? For example, should I have known how to generate a list like that?

**Comment by J0WI** [comment]
You should have a look at https://github.com/EFForg/https-everywhere/blob/master/ruleset-style.md and https://github.com/EFForg/https-everywhere/blob/master/ruleset-testing.md

**Comment by bgkonzman** [comment]
Thanks for pointing me at those docs.

Looking at some other rulesets, it seemed common practice to create a separate file for mismatches. I went ahead and did that for the blog, status, and career subdomains. If you'd prefer I not do that, let me know and I'll remove the file and update the comments at the top of Udacity.com.xml, or whatever else you suggest.

Again, thanks for the patience, help, and reviews.

**Comment by J0WI** [comment]
Perfect!

**Pull request merged**